python-api/gstswitch/controller.py

```diff
@@ -154,6 +162,28 @@ def establish_connection(self):
             default_interface=self.default_interface)
 
         self.connection.connect_dbus()
+        self.connection.signal_subscribe(self.cb_signal_handler)
+
+    # pylint: disable=star-args
+    def cb_signal_handler(self, connection, sender_name, object_path,
```

Comment:
yes, but pylint has a special rule for methods starting with cb_ -- this is why I changed it in the first place: http://docs.pylint.org/faq.html#i-have-a-callback-function-where-i-have-no-control-over-received-arguments-how-do-i-avoid-getting-unused-argument-warnings